src/combat.cpp

```diff
@@ -843,15 +843,23 @@ void Combat::doTargetCombat(Creature* caster, Creature* target, CombatDamage& da
 	}
 
 	Player* casterPlayer = caster ? caster->getPlayer() : nullptr;
+	Player* targetPlayer = target ? target->getPlayer() : nullptr;
```

Comment:
do we need `: nullptr` in these?

Reply:
yes, how would you go with removing nullptr?
target->getPlayer() would crash if target was nullptr (target should never be though, but with caster it might be true)